pkg/ctl/create/utils.go

```diff
@@ -52,6 +53,18 @@ func checkVersion(cmd *cmdutils.Cmd, ctl *eks.ClusterProvider, meta *api.Cluster
 	return nil
 }
 
+func showDevicePluginMessageForInstance(instanceType string, instancesDistribution *api.NodeGroupInstancesDistribution) {
```

Comment:
is this PR having dependency with https://github.com/weaveworks/eksctl/pull/2057 ?